samples/mp4-decode/index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>WebCodec MP4 frame extration demo</title>
+  <meta http-equiv="origin-trial" content="Aje8D+7PG97BFcl3a9WSQvpbg81wM2U1tqT5a6fBp/eikjy/c/sJiZIb8grxqxpFJdeV8Xsujz8+oTBeHQZK5gUAAABdeyJvcmlnaW4iOiJodHRwczovL3RndWlsYmVydC1nb29nbGUuZ2l0aHViLmlvOjQ0MyIsImZlYXR1cmUiOiJXZWJDb2RlY3MiLCJleHBpcnkiOjE2MDgwNTgwMTJ9" />
+</head>
+<body>
+  <p>
+    This demo extracts all frames from an MP4 file and renders them to a canvas as fast as possible. It uses <a href="https://github.com/gpac/mp4box.js/">mp4box.js</a> to parse and demux the file.
+  </p>
+  <p>
+    Frames extracted: <span id="frameCount"></span>
+  </p>
+  <canvas></canvas>
+
+<script src="mp4box.all.min.js"></script>
```

Comment:
Move script out of body or otherwise fix indenting?

Reply:
Done (in #219)